core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -914,7 +915,8 @@ struct PolicyPropertyAdaptor<WorkItemProperty::ImplWorkItemProperty<P>,
                   typename policy_in_t::traits::index_type,
                   typename policy_in_t::traits::iteration_pattern,
                   typename policy_in_t::traits::launch_bounds,
-                  WorkItemProperty::ImplWorkItemProperty<P>>;
+                  WorkItemProperty::ImplWorkItemProperty<P>,
+                  typename policy_in_t::traits::occupancy_control>;
```

Comment:
Non-blocking note: this drops the `is_graph_kernel` property on the floor, but since we are the ones adding that and not the user, and since it always gets added last, that shouldn't be a problem. We need to re-work the whole policy properties infrastructure anyway, so don't worry about it for now.

Reply:
https://github.com/kokkos/kokkos/pull/3360#pullrequestreview-484552137

Reply:
Dammit I had not used a permanent URL.  Anyway I highlighted the `PolicyPropertyAdaptor` code at the time.  I would rather not fix it here.